Makefile

```diff
@@ -45,10 +45,14 @@ db/setup:
 
 db/migrate:
 	bee migrate -driver=postgres -conn="$(DATABASE_URL)"
+	pg_dump $(DATABASE_URL) --schema-only -f ./database/schema.sql
```

Comment:
Use the schema file to init the Postgres database followed [the document about Initialization scripts on Postgres docker image](https://hub.docker.com/_/postgres)